packages/styles/scss/components/radio-button/_radio-button.scss

```diff
@@ -109,16 +110,7 @@ $radio-border-width: 1px !default;
       transform: scale(0.5);
 
       // Allow the selected button to be seen in Windows HCM for IE/Edge
-      @media screen and (-ms-high-contrast: active) {
-        // Utilize a system color variable to accommodate any user HCM theme
-        background-color: WindowText;
-      }
-
-      // Firefox only HCM solution
-      @media screen and (prefers-contrast) {
-        // Utilize a system color variable to accommodate any user HCM theme
-        border: 2px solid WindowText;
-      }
+      @include high-contrast-mode('icon-fill');
```

Comment:
Can we add this and the mixin updates to the V10 package as well?

Reply:
@tw15egan should i do this in the new fangled v10 branch instead now that @tay1orjones has made that?

Reply:
@dakahn Yeah good call, I vote for opening that in a separate PR into the `v10` branch

Reply:
@dakahn @tay1orjones @tw15egan has this been delivered to the v10 branch?  If so, can you include a link to that PR?

We're failing a particular WCAG guideline wrt Text Contrast as a result of this issue and our team does not plan to upgrade to v11 in the near term so having this fix in v10 would go a long way.  Thanks.

Reply:
@fbarroso24 It has not. Thanks for letting us know, we can get this into the v10 branch and cut a release.

Reply:
@fbarroso24 This has been ported to the v10 branch via https://github.com/carbon-design-system/carbon/pull/11330, and is available in [v10.57.1](https://github.com/carbon-design-system/carbon/releases/tag/v10.57.1)